### PR TITLE
mb_convert_kana throws ValueError when impossible combination since PHP 8.2

### PR DESCRIPTION
**Pull request by youkidearitai**
Throws ValueError since PHP 8.2.
I add ChangeLog section it.

```php
mb_convert_kana("abc", "sS", "UTF-8");

Fatal error: Uncaught ValueError: mb_convert_kana(): Argument #2 ($mode) must not combine 'S' and 's' flags in /in/gvk4D:3
Stack trace:
#0 /in/gvk4D(3): mb_convert_kana('abc', 'sS', 'UTF-8')
#1 {main}
  thrown in /in/gvk4D on line 3
```

3v4l <https://3v4l.org/gvk4D>

### CODE REVIEW

**Comment by mumumu** [comment]
Strict validation of conversion flags to mb_convert_kana, as of PHP 8.2.0 RC1.
https://github.com/php/php-src/commit/dcaa010fffb23af2fc0fdd20dc3c00d56ca34f3f

**Comment by mumumu** [comment]
Better to add "errors" section. (example in [mb_encoding_aliases](https://github.com/php/doc-en/blob/1a025802f7201d353c1c1c4dccc7d9d6fdf563d4/reference/mbstring/functions/mb-encoding-aliases.xml#L41-L47) )
```
 <refsect1 role="errors">
  &reftitle.errors;
  <para>
   <classname>ValueError</classname> is thrown if the 
   combination of <parameter>mode</parameter> is invalid.
  </para>
 </refsect1>
```

File: reference/mbstring/functions/mb-convert-kana.xml [hunk 1]
```diff
@@ -176,6 +176,10 @@
      </row>
     </thead>
     <tbody>
+      <row>
+        <entry>8.2.0</entry>
+        <entry>Throw ValueError when impossible combinations (ex: "sS" throws ValueError)</entry>
```
Comment (mumumu) [inline reference/mbstring/functions/mb-convert-kana.xml:181]:
```suggestion
        <entry>
          <classname>ValueError</classname> is thrown if the 
   combination of <parameter>mode</parameter> is invalid. (ex: "s" and "S")
        </entry>
```

Comment (youkidearitai) [reply reference/mbstring/functions/mb-convert-kana.xml]: Thanks. Fixed.

**Comment by youkidearitai** [comment]
> Better to add "errors" section. (example in [mb_encoding_aliases](https://github.com/php/doc-en/blob/1a025802f7201d353c1c1c4dccc7d9d6fdf563d4/reference/mbstring/functions/mb-encoding-aliases.xml#L41-L47) )

Thank you very much! Added.

**Comment by mumumu** [comment]
@youkidearitai Hey. CI has failed because your fix has training whitespace. You can fix it by removing line break of the following line, maybe.

https://github.com/php/doc-en/pull/2647/files#diff-0812e8600243b5d641ce89b0d68ac0ddc7e98691a9b75ac6e5dafb19a2bd6738R191

**Review by mumumu — APPROVED** [review]
LGTM

File: reference/mbstring/functions/mb-convert-kana.xml [hunk 2]
```diff
@@ -176,6 +185,13 @@
      </row>
     </thead>
     <tbody>
+      <row>
+        <entry>8.2.0</entry>
+        <entry>
+          <classname>ValueError</classname> is thrown if the
+   combination of <parameter>mode</parameter> is invalid. (ex: "s" and "S")
```
Comment (Girgias) [inline reference/mbstring/functions/mb-convert-kana.xml:192]:
```suggestion
   A <classname>ValueError</classname> is now thrown if the
   combination of different <parameter>mode</parameter>s is invalid.
```

**Comment by Girgias** [comment]
Welp, that's unfortunate lol

**Pull request merged**